bin/start.js

```diff
@@ -34,7 +40,14 @@ browserSync.init({
                 `${CONFIG.sourceDir}/**/*`
             ],
             fn: () => {
-                Build.buildSync();
+                if (timeSinceDebounce(devDebounceMarker) > 5) {
```

Comment:
This isn't true debouncing, though. This will run for the _first_ marker, but not the any subsequent markers.
 A true debounce will wait until there are no events within the timeframe after the _last_ marker.

Could we use `_.debounce()`? https://lodash.com/docs/4.17.4#debounce

Reply:
Ok, so this will fire it twice then if you make multiple edit+saves in a file within the `n`-second window -- once the moment the function bounces, and again once the function has finished bouncing.

Reply:
I made sure that the debounce fires `trailing: true` in the options for the lodash call.